INA-6986: Update property fields page with new text array type (#27680)
DataDog/documentation: content/en/service_management/incident_management/incident_settings/property_fields.md

### Field types

In addition to the five default fields and the fields based on metric tags, you can also create custom property fields and mark them as required at the creation of an incident. There are five kinds of custom fields you can create:

**Single-Select**

> : A dropdown field that can only have one value assigned at a time per incident. Values can be predefined in-line from the UI or by uploading values through a CSV file.

Reviewer: ```suggestion
: A dropdown field that can only have one value assigned at a time per incident. You set the available values when defining the field.
```

**Multi-Select**

> : A dropdown field that can have multiple values assigned per incident. Values can be predefined in-line from the UI or by uploading values through a CSV file.

Reviewer: ```suggestion
: A dropdown field that can have multiple values assigned per incident. You set the available values when defining the field.
```

**Text Array**

> : A free-form field that can have multiple values assigned per incident. Values are not predefined and can be assigned at any point during incident response.

Reviewer: ```suggestion
: A free-form field that can have multiple values assigned per incident. Incident responders define arbitrary values when updating the field on an incident.
```
